Userland/Games/Solitaire/Game.cpp

```diff
@@ -231,6 +231,7 @@ void Game::keydown_event(GUI::KeyEvent& event)
     } else if (event.shift() && event.key() == KeyCode::Key_F11) {
         dump_layout();
     }
+    event.ignore();
```

Comment:
I think this should be in an `else` branch. "Accepted" means that the widget did something meaningful with the event and no further processing is needed.